templates/include/telemetry/environment.1.schema.json

```diff
@@ -656,6 +656,12 @@
                     "string",
                     "null"
                   ]
+                },
+                "type": {
+                  "type": [
+                    "string",
+                    "null"
+                  ]
```

Comment:
It looks like this can be an enumerated type, i.e.

```json
enum: ["SSD", "HDD"]
```

Reply:
Thanks for pointing this out! Updated the PR. 

I kept the `type` as 
```
"type": [
  "string",
  "null"
]
```

Would `null` still work?

Reply:
`null` values should still be valid because nothing has changed other than limiting the the values of `string` to the values contained in `enum`. If you aren't sure, I would encourage you to explicitly write another validation test that includes null (and missing!) values for the new column you've added.

Reply:
Thanks! Added two more tests, for null and empty string.